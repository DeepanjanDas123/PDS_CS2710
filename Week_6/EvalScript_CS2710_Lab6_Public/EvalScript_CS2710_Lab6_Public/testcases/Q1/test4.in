12
NEW PUSH_BEGIN 10 PUSH_END 20 PUSH_BEGIN 5 PUSH_END 30 PUSH_BEGIN 3 DELETE 20 DELETE 5 PUSH_END 40 PUSH_BEGIN 1 DELETE 30 PUSH_END 50