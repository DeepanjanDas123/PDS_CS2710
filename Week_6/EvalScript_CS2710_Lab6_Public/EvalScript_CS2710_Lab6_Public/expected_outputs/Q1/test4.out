1 3 10 40 50
